api/modelss/build-log.js

```diff
@@ -5,19 +5,50 @@ const associate = ({ BuildLog, Build }) => {
   })
 }
 
+const beforeValidate = (buildLog) => {
+  return saniztizeBuildSecrets(buildLog)
+}
+
+const saniztizeBuildSecrets = (buildLog) => {
```

Comment:
saniztizeBuildSecrets -> sanitizeBuildSecrets

I know this is from the original, but might as well fix typos on the way...